content/sql/concepts/math-functions/terms/log/log.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: 'LOG()'
+Description: 'Returns the natural logarithm of a given number.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'SQLite'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'Functions'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+The **`LOG()`** function in SQL returns the natural logarithm of a given number, or the logarithm of the number to the specified base. The given number must be greater than 0, and the base must be greater than 1.
+
+## Syntax
+
+```sql
```

Comment:
I'd like a sentence after the code block explaining what the variables `number` and `base` are. I also think it might be good to have this in a ` ```pseudo ` block with just the functions outside a `SELECT` statement, as they aren't just legal in the `SELECT` clause, we can also put them in a `WHERE` or `HAVING` clause.

Reply:
Updated, please check, thanks. @SSwiniarski